src/LaddaButton.jsx

```diff
@@ -74,6 +74,8 @@ class LaddaButton extends Component {
     if (props.loading !== this.props.loading) {
       if (props.loading) {
         this.laddaInstance.start()
+      } else if (props.disabled) {
+        this.laddaInstance.disable()
```

Comment:
Also, leave a comment that describes why disable has to be set and link to https://github.com/hakimel/Ladda/blob/0218e92c78bda78931292905cfc332d0b01fe13c/js/ladda.js#L117 for context.